internal/tlsprofile/options.go

```diff
@@ -17,3 +21,19 @@ type TLSProfileOptions struct {
 func (o TLSProfileOptions) TLSCipherSuites() string {
 	return strings.Join(o.Ciphers, ",")
 }
+
+// MinVersionShort returns the min TLS version but only the number instead of VersionTLS10 it will return 1.0.
+func (o TLSProfileOptions) MinVersionShort() string {
+	switch o.MinTLSVersion {
+	case string(openshiftconfigv1.VersionTLS10):
+		return "1.0"
+	case string(openshiftconfigv1.VersionTLS11):
+		return "1.1"
+	case string(openshiftconfigv1.VersionTLS12):
+		return "1.2"
+	case string(openshiftconfigv1.VersionTLS13):
+		return "1.3"
+	default:
+		return ""
```

Comment:
Would it make sense to return an error saying "unsupported version" or something?